content/docs/0.18.x/define/quality-gates/index.md

```diff
@@ -1,24 +1,131 @@
+weight: 70
 keywords: [0.18.x-quality_gates]
 ---
 
+A quality gate allows you to validate a deployment or release
+using data delivered by your observability platform
+and using quality criteria that you define as Service Level Objectives (SLOs).
+
+:information_source: If you are not familiar with the concept of a quality gate, learn more [here](../../../concepts/quality_gates). 
+
+## Definition of project, stage, and service
+
+Let's assume you have an application that is running in an environment and composed of one or multiple services (aka. microservices). For example, you have an application called `easyBooking`, which can be broken down into the `booking` and `payment` service. Besides, the application is running in a `quality_assurance` environment (aka. stage).
+In order to manage your application and services in Keptn, you need a Keptn [project, stage](../../manage/project/) and [services](../service).
+For the `easyBooking` application, the Keptn entities of a project, stage, and service map to the example as follows:
+
+* `project`: *easyBooking*
+* `stage`: *quality_assurance*
+* `service`: *booking* & *payment* (For the sake of simplicity, a quality gate will be configured for `booking` only.)
+
+For defining the stage(s) a service has to go through, a [Shipyard](../../reference/files/shipyard) file is needed.
+Since a quality gate should be configured for the *quality_assurance* environment only, the corresponding Shipyard for the *easyBooking* project looks as follows:
+
+```yaml
+apiVersion: "spec.keptn.sh/0.2.3"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "quality-assurance"
+```
+
+<details><summary>*Why I do not need to specify a task sequence for quality gates in this stage?*</summary>
+<p>
+
+You do not have to define any task sequence in a stage because quality gates (aka. `evaluation`) are a built-in Keptn capability. Hence, there is no need to explicitly add an `evaluation` sequence. However, the explicit form of the above Shipyard file would look as the following one, which behaves the same way: 
+
+```yaml
+apiVersion: "spec.keptn.sh/0.2.3"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "quality-assurance"
+      sequences:
+       - name: "evaluation"
+         tasks:
+         - name: "evaluation"
+```
+
+</p>
+</details>
+
+## Create project and service
+
+* To create the Keptn project `easyBooking`,
+use the [keptn create project](../../reference/cli/commands/keptn_create_project/) command
+as discussed in [Create a project](../../manage/project).
+
+* To create the Keptn service `booking`,
+use the [keptn create service](../../reference/cli/commands/keptn_create_service/) command
+as discussed in [Create a service](../service/#create-a-service).
+
+**Note:** It is not necessary to create the stage since it is declared in the Shipyard that is applied during the project creation.
```

Comment:
Should we also mention somewhere that stages can not be added or removed after the initial creation?

Reply:
Yes, that is mentioned in several places.  Actually, I think all of this basic create-service text that is in the Quality Gates page needs to be deleted and replaced with a link to the "Create a project" page under "Managing Projects."   After we get the files restructured, we can look and see where we need to reiterate that stages cannot be added or removed.